app/assets/javascripts/views/data_portal/filters/ApplyFiltersView.js

```diff
@@ -0,0 +1,69 @@
+(function (App) {
+
+  App.View.ApplyFiltersView = Backbone.View.extend({
+
+    template: JST['templates/data_portal/filters/apply-filters'],
+
+    initialize: function (options) {
+      // clones array with an object array inside
+      this._indicators = options.indicators.map(function (indicator) {
+        return {
+          name: indicator.name,
+          id: indicator.id,
+          options: Array.prototype.slice.call(indicator.options, 0)
+        };
+      });
+
+      // sorts indicator array by indicator name
+      this._indicators.sort(function (a, b) {
+        if (a.name >= b.name) {
```

Comment:
This condition is wrong, it should be `a.name > b.name`. Otherwise, you'll never return `0` which is the value for equality.